pytorch_impl/datareader.py

```
@@ -0,0 +1,403 @@
+        self.target_is_hap = False
+        self.ref_n_header_lines = []
+        self.ref_n_data_header = ""
+        self.target_n_header_lines = []
+        self.target_n_data_header = ""
+        self.ref_separator = None
+        self.delimiter_dictionary = {"vcf": "\t", "csv": ",", "tsv": "\t", "infer": "\t"}
+        self.ref_file_extension = "vcf"
+        self.test_file_extension = "vcf"
+        self.target_is_phased = True
+
+    def __read_csv(self, file_path: str, is_vcf: bool = False, is_reference: bool = False,
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)